packages/ui/package.json

```diff
@@ -43,6 +43,10 @@
     "url": "https://github.com/hazelcast/frontend-shared/issues"
   },
   "dependencies": {
+    "@codemirror/view": "0.18.8",
+    "@codemirror/basic-setup": "0.18.0",
+    "@codemirror/legacy-modes": "0.18.0",
```

Comment:
Should this one be a dev dependency?

Reply:
@aigoncharov no, why would be want that? it's just convenient to have it there instead of your app code. (see the unresolved comments at the top.)